ls1intum/Orion
# [Bugfix] Prevent Crashes From Unsupported Programming Languages

### Motivation and Context
Fixes #58 

### Description
Updates Orions ProgrammingLanguage enumeration to the latest version so no further parsing errors should occur. Since Orion still only really supports java (and traces of python), a warning will be shown for all other languages to inform the user they will have to configure the project manually.

### Steps for Testing
1.  Install the release as described in [the readme](https://github.com/ls1intum/Orion/blob/master/README.md#testing-of-pull-requests)
2.  Download exercises in various languages as various roles (as student, instructor and tutor)
3.  Java should work properly
4.  All other languages (except Python, which sets its sdk automatically) should show a warning
